src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5750,6 +5750,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_MissingNonNullTypesContextForAnnotationInGeneratedCode_Title" xml:space="preserve">
     <value>The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source.</value>
   </data>
+  <data name="WRN_NullReferenceInitializer" xml:space="preserve">
+    <value>Object or collection initializer implicitly dereferences possibly null reference member '{0}'.</value>
```

Comment:
>reference [](start = 82, length = 9)

Should probably just be `member`, it could be an `int?` which isn't a reference member.

Reply:
I think this scenario is not possible for this form of initialization because the System.Nullable is immutable

Reply:
For a program like this:

```cs
using System;

public struct S {
    public C c;
}

public class C {
    public S? s = null;
    
    
    public static void M() {
        new C
        {
            s = { Value = { c = { s = null } } }
        }
    }
}
```
It looks like the actual diagnostic is: `error CS1918: Members of property 'S?.Value' of type 'S' cannot be assigned with an object initializer because it is of a value type`

Reply:
I am however inclined to change the wording at this point at least for economy of language though.